File: codes/python/chap06/answer_A37.py
# 入力
N, M, B = map(int, input().split())
A = list(map(int, input().split()))
C = list(map(int, input().split()))

# 答えの計算
Answer = 0
for i in range(N):
	Answer += A[i] * M
Answer += B * N * M
for j in range(M):
	Answer += C[j] * N

# 出力
print(Answer)

File: codes/cpp/chap06/answer_A37.cpp
#include <iostream>
using namespace std;

long long N, M, B;
long long A[200009], C[200009];

int main() {
	// 入力
	cin >> N >> M >> B;
	for (int i = 1; i <= N; i++) cin >> A[i];
	for (int j = 1; j <= M; j++) cin >> C[j];

	// 答えの計算
	long long Answer = 0;
	for (int i = 1; i <= N; i++) Answer += A[i] * M;
	Answer += B * N * M;
	for (int j = 1; j <= M; j++) Answer += C[j] * N;

	// 出力
	cout << Answer << endl;
	return 0;
}
